brave-intl/publishers
# Improve notes system 🗒

##  Improve notes system

### Notes section

#### Features

* Changed look and feel
* HTML tags will be displayed (javascript won't)
* New @mention system 
* Reply system
* Ability to edit and delete comments

#### How To Test

1. Go to a publisher
2. Create a few notes
3. Reply to those notes
4. Delete em / edit em / whatevs
5. Everything should work fine 🎉
